nsv13/code/game/objects/structures/catwalk.dm

```diff
@@ -38,13 +40,25 @@
 	if(hatch_open)
 		icon_state = "catwalk_hatch_open"
 		obj_flags &= ~(BLOCK_Z_OUT_DOWN | BLOCK_Z_IN_UP) //Minecraft trap door.
+		flags_1 &= ~PREVENT_CLICK_UNDER_1
 		smooth = SMOOTH_FALSE
 		clear_smooth_overlays()
+		queue_smooth(src)
 	else
-		icon_state = initial(icon_state)
+		icon_state = ""
 		obj_flags |= (BLOCK_Z_OUT_DOWN | BLOCK_Z_IN_UP)
+		flags_1 |= PREVENT_CLICK_UNDER_1
 		smooth = SMOOTH_TRUE
 		queue_smooth(src)
 
-/obj/structure/lattice/catwalk/over/ship/can_lay_cable()
-	return hatch_open
+
+/obj/structure/lattice/catwalk/over/ship/deconstruct()
+	new plating_type(get_turf(src))
+	. = ..()
+
+/obj/structure/lattice/catwalk/over/attackby(obj/item/C, mob/user, params)
+	if(resistance_flags & INDESTRUCTIBLE)
+		return
+	if(C.tool_behaviour == TOOL_WIRECUTTER)
+		to_chat(user, "<span class='notice'>Slicing [name] joints ...</span>")
+		deconstruct()
```

Comment:
```suggestion
/obj/structure/lattice/catwalk/over/attackby(obj/item/C, mob/user, params)
	if(resistance_flags & INDESTRUCTIBLE)
		return ..()
	if(C.tool_behaviour == TOOL_WIRECUTTER)
		to_chat(user, "<span class='notice'>Slicing [name] joints ...</span>")
		deconstruct()
	else
		return ..()

```
Still needs to be able to get attacked.
Cable laying is done with can_lay_cable() so  (in [code/game/turfs/turf.dm](https://github.com/BeeStation/NSV13/blob/5232787ff4630796e07e1099373ea4af669ae032/code/game/turfs/turf.dm#L335))

Reply:
Regular catwalks, including the `/over/` type, override attackby() just like that and it's fine.
No idea what you are talking about with can_lay_cable(), cable laying works just fine in every situation I tested.

Reply:
Alright then ig, if it works it works